docs/docs/components/reanimated_swipeable.md

```diff
@@ -81,27 +81,29 @@ an argument.
 
 ### `renderLeftActions`
 
-method that is expected to return an action panel that is going to be revealed from the left side when user swipes right.
-This map describes the values to use as inputRange for extra interpolation:
+accepts a function which receives the following arguments:
 
-SharedValue: [startValue, endValue]
+- `progress` value representing swiping progress relative to the width of the returned element.  
+  Equals `0` when `swipeable` is closed, `1` when left action is open.  
```

Comment:
```suggestion
  Equals `0` when `swipeable` is closed, `1` when `swipeable` is open.  
```

Reply:
`swipeable` cannot be open, either the `left action` is open, or the `right action` can be open.

Reply:
> `swipeable` cannot be open

Why is that? For example, it has `onSwipeableWillOpen` and `onSwipeableOpen` props. What I would change here is I'd use passive voice, i.e.:

```suggestion
Equals `0` when `swipeable` is closed, `1` when `swipeable` is opened. 
```

Reply:
`onSwipeableWillOpen` distinguishes between `swipeable` opening it's left and right `element`.
Here, the distinction of which side is open is equally necessary.

What about something like this?

> Equals `0` when `swipeable` is closed, `1` when `swipeable` has its left `element` open.

or

> Equals `0` when `swipeable` is closed, `1` when the left `element` is open.

---

`element` could be replaced with `action`, but I think you mentioned you'd prefer to avoid `action` [here](https://github.com/software-mansion/react-native-gesture-handler/pull/3121#discussion_r1782573527), was that a context-specific preference or a general rule?

Reply:
> Here, the distinction of which side is open is equally necessary.

Isn't it implied by the fact that we're in `renderLeftActions` section?

Reply:
> `onSwipeableWillOpen` distinguishes between `swipeable` opening it's left and right element.

Yes it does, but the name indicates that `Swipeable` itself will be opened.  Why can't you say that it is _**opened**_ if you can say that it is _**closed**_? 

>`element` could be replaced with `action`

I think the best we could replace it with would be `action panel`, as in `tsdocs`. But still, I believe that `Swipeable` fits better here. (cc @j-piasecki, @kacperkapusciak)

Reply:
I agree, if we want to avoid using terms like left/right action, since these are already implied by the function name, then standalone `swipeable` definitely fits best.

Fixed in [df0ff8c](https://github.com/software-mansion/react-native-gesture-handler/pull/3121/commits/df0ff8cb7be3b0c651b048dced53ffa8b4731b42) throughout [9845471](https://github.com/software-mansion/react-native-gesture-handler/pull/3121/commits/9845471eab4539a1a6d9f4164ab3483a159b296e)